specs/lib/mergeDuplicateMaterialsSpec.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+var mergeDuplicateMaterials = require('../../lib/mergeDuplicateMaterials');
+
+describe('mergeDuplicateMaterials', function() {
+    it('merges duplicate materials', function() {
+        var gltf = {
+            meshes : {
+                meshOne : {
+                    primitives : [
+                        {
+                            material : 'materialOne',
+                        },
+                        {
+                            material : 'materialThree'
+                        }
+                    ]
+                },
+                meshTwo : {
+                    primitives : [
+                        {
+                            material : 'materialTwo'
+                        },
+                        {
+                            material : 'materialThree'
+                        }
+                    ]
+                }
+            },
+            materials : {
```

Comment:
I'm not sure that's necessary. This stage is designed to merge duplicates, we have other stages for `removeUnusedXXX` that we already have tests for.